test_engine/terraform/aws/ubuntu/user-data-scripts/provision.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash 
+
+DOCKER_VERSION=20.10
+
+sudo apt-get update 
+sudo apt-get install -y build-essential git nfs-common
+
+until (curl https://releases.rancher.com/install-docker/${DOCKER_VERSION}.sh | sudo sh); do
```

Comment:
It seems we have many places using this way to install docker, but I feel we can just rely on the host package (like apt install docker?) to install docker instead to make things simple. WDYT?

This can be improved for other places later as well. 

cc @yangchiu